Source/WebCore/css/typedom/CSSNumericValue.cpp

```diff
@@ -333,8 +333,8 @@ bool CSSNumericValue::equals(FixedVector<CSSNumberish>&& values)
 {
     // https://drafts.css-houdini.org/css-typed-om/#dom-cssnumericvalue-equals
     auto numericValues = WTF::map(WTFMove(values), rectifyNumberish);
-    return WTF::allOf(numericValues, [&] (const Ref<CSSNumericValue>& value) {
-        return this->equals(value.get());
+    return std::ranges::all_of(numericValues, [protectedThis = Ref { *this }] (const Ref<CSSNumericValue>& value) {
+        return protectedThis->equals(value.get());
```

Comment:
WTF::allOf includes a `NOESCAPE` annotation that eliminates the need for this refcount. Style improvements are nice, but only when they achieve equal functionality.

Reply:
But... why? Sure, you can replace uses of allOf with loops, but the purpose of allOf is not to have to write out a loop.

If std::ranges::all_of is less capable than WTF::allOf, then we should not remove WTF::allOf.